reference/excel/chartcollection.md

```diff
@@ -43,6 +43,12 @@ chartCollectionObject.add(type, sourceData, seriesBy);
 |sourceData|object|The Range object corresponding to the source data.|
 |seriesBy|string|Optional. Specifies the way columns or rows are used as data series on the chart.  Possible values are: Auto, Columns, Rows|
 
+**Below are the valid chart types**:
```

Comment:
Please change this heading to "Valid chart type values". For readability, consider making a bulleted list, or at a minimum alphabetizing the list of values. Also, in the Description column for the type parameter, remove the sentence that begins with "Possible values are..." and replace with a reference to the list: "For a list of possible values, see Valid chart type values."